subvol-mount-generator/suse-btrfs-subvol-mount-generator

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+output="${1:?}"
+
+declare -A mountpoints
+for i in /var /usr/local /srv /root /opt /home /.snapshots; do mountpoints["$i"]=""; done
+
+generate_subvol_mounts() {
+	root_uuid=$(findmnt / -r -n -o UUID)
+	quoted_uuid="${root_uuid//-/\\x2d}"
```

Comment:
The correct way to quote this is to use `systemd-escape -p /dev/disk/by-uuid/...`

Reply:
didn't know about that one., thanks!

Reply:
Not changed yet

Reply:
now it is